libdnf/hy-util.cpp

```diff
@@ -139,6 +139,10 @@ hy_detect_arch(char **arch)
         strcpy(un.machine, "mipsel");
     else if (!strcmp(un.machine, "mips64"))
         strcpy(un.machine, "mips64el");
+#endif
+#ifdef __LOONGARCH__
+    if (!strcmp(un.machine, "loongarch64"))
+        strcpy(un.machine, "loongarch64");
```

Comment:
What is the purpose of this code?
If `un.machine` contains the string" loongarch64 ", write it there again. Why?

Reply:
Sorry, this code is redundant，code has been deleted